cmd/lncli/commands.go

```diff
@@ -112,11 +112,9 @@ func actionDecorator(f func(*cli.Context) error) func(*cli.Context) error {
 			// WalletUnlocker server active) and most likely this
 			// is because of an encrypted wallet.
 			if ok && s.Code() == codes.Unimplemented {
-				return fmt.Errorf("Wallet is encrypted. " +
-					"Please unlock using 'lncli unlock', " +
-					"or set password using 'lncli create'" +
-					" if this is the first time starting " +
-					"lnd.")
+				return fmt.Errorf("Wallet is unlock. " +
```

Comment:
This is incorrect. You should instead add `|| c.Command.Name == "changepassword"` to the `if` statement above.

Reply:
Should I still leave the error message? Or revert it?

Reply:
Revert it. This error message is for the opposite case, when you try to call "GetInfo" or another command that is not available when the wallet is locked.